packages/react-search-ui/README.md

```diff
@@ -400,3 +1010,190 @@ function ClearFilters({ filters, clearFilters }) {
+| `search` | `searchTerm` - String    | [ResultList](#resultlist) | The search string to query on.                                                                                                                                                                                                                                                                                                                              |
+|          | `searchOptions` - Object |                           | `searchOptions` follow the format from App Search's [Search API](https://swiftype.com/documentation/app-search/api/search). The following properties are supported:<br/>- facets<br/>- filters<br/>- result_fields<br/>- search_fields<br/>- sorting <br/> <br/> Additionally, we will pass:<br/> - disjunctiveFacets<br />- disjunctiveFacetsAnalyticsTags |
+
+<a id="resultlist"></a>
+
+#### ResultList
+
+A ResultList object wraps the `results` object from a [Response](#appsearchresponse).
+
+| field        | type                             | description                                                                             |
+| ------------ | -------------------------------- | --------------------------------------------------------------------------------------- |
+| `rawResults` | Array[Object]                    | Raw `results` field from a [Response](#appsearchresponse).                              |
+| `results`    | Array[[ResultItem](#resultItem)] | The `results` array, mapped to a list of [ResultItem](#resultItem)s.                    |
+| `info`       | Object                           | All of the fields from a response other than `results`. This means `meta` and `facets`. |
+
+<a id="resultItem"></a>
+
+#### ResultItem
+
+Each Result Item is a wrapped around an individual result, in the
+
+| field        | type             | description                                                              |
+| ------------ | ---------------- | ------------------------------------------------------------------------ |
+| `data`       | Object           | Raw object from the `results` array on a [Response](#appsearchresponse). |
+| `getRaw`     | Function(String) | Convenience function for getting `raw` field data.                       |
+| `getSnippet` | Function(String) | Convenience function for getting `snippet` field data.                   |
+
+<a id="appsearchresponse"></a>
+
+#### Search API response example
+
+More details can be found in the
+[App Search Search API documentation](https://swiftype.com/documentation/app-search/api/search).
+
+For `results`, each result can have one of two fields: `raw` and `snippet`. The
```

Comment:
We have a guide for this https://swiftype.com/documentation/app-search/guides/sanitization -- just an FYI in case you have not seen 'er.

Reply:
Perfect!